actioner/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "22996056-6abc-4d33-be11-3e27626b8a26",
+  "app_id": "actioner",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Alternative way to connect your Datadog account with Actioner",
+    "title": "actioner",
+    "media": [],
+    "classifier_tags": [
+      "Queried Data Type::Incidents",
+      "Queried Data Type::Monitors",
+      "Queried Data Type::Teams"
```

Comment:
Is your integration only pulling data out of Datadog? Please include submitted data type if submitting data into Datadog. Additionally, the queried data types: Monitors, and teams are not supported data types. Please make sure you have included the classifier tags that are in this [list](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags) including support OS, associated offering type, associated categories, and supported query and submitted data types.

Reply:
The mere purpose of the integration is to enable users to make OAuth2 connection. So, Actioner neither publishes nor fetch data by itself. Rather, users can define workflows in which Datadog API calls are made.

Reply:
Please include categories